2021/09/18 01:11:50 Starting App Insight Logger for task:  runTaskLet
2021/09/18 01:11:50 Version: 3.0.01719.0004 Branch: .SourceBranch Commit: a462f58
2021/09/18 01:11:50 Attempt 1 of http call to http://10.0.0.6:16384/sendlogstoartifacts/info
2021/09/18 01:11:50 Send process info logs to master server succeeded
2021/09/18 01:11:50 Attempt 1 of http call to http://10.0.0.6:16384/sendlogstoartifacts/status
2021/09/18 01:11:50 Send process info logs to master server succeeded
[2021-09-18T01:11:50.198290] Entering context manager injector.
[2021-09-18T01:11:50.656762] context_manager_injector.py Command line Options: Namespace(inject=['ProjectPythonPath:context_managers.ProjectPythonPath', 'RunHistory:context_managers.RunHistory', 'TrackUserError:context_managers.TrackUserError', 'UserExceptions:context_managers.UserExceptions'], invocation=['python train.py --dataset azureml:credit-dataset:1'])
Script type = COMMAND
[2021-09-18T01:11:50.661006] Command=python train.py --dataset azureml:credit-dataset:1
[2021-09-18T01:11:50.661266] Entering Run History Context Manager.
[2021-09-18T01:11:52.230608] Command Working Directory=/mnt/batch/tasks/shared/LS_root/jobs/canaryws/3dcd96adcf1a41f1bccd5dc51fdeb476/3744b61c-6723-498b-83f3-018dd8df3160/wd/azureml/3744b61c-6723-498b-83f3-018dd8df3160
[2021-09-18T01:11:52.230665] Starting Linux command : python train.py --dataset azureml:credit-dataset:1
####1:  ['azureml-setup', 'train.py', 'main.py', 'azureml_compute_logs', 'outputs', 'logs', 'dataset', 'azureml-logs']
####2:  ['dataset/german_credit_data.csv']
###3
training accuracy: 0.745
test data accuracy: 0.735
[2021-09-18T01:11:53.234081] Command finished with return code 0


[2021-09-18T01:11:53.234392] The experiment completed successfully. Finalizing run...
Cleaning up all outstanding Run operations, waiting 900.0 seconds
1 items cleaning up...
Cleanup took 0.057724952697753906 seconds
[2021-09-18T01:11:53.424598] Finished context manager injector.
2021/09/18 01:11:54 Attempt 1 of http call to http://10.0.0.6:16384/sendlogstoartifacts/status
2021/09/18 01:11:54 Send process info logs to master server succeeded
2021/09/18 01:11:54 Not exporting to RunHistory as the exporter is either stopped or there is no data.
Stopped: false
OriginalData: 3
FilteredData: 0.
2021/09/18 01:11:54 Process Exiting with Code:  0
2021/09/18 01:11:55 All App Insights Logs was sent successfully or the close timeout of 10 was reached
